client/src/lib/components/SearchInput/SearchInput.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+  import type { SelectOption } from "src/definitions/form";
+
+  import { createEventDispatcher } from "svelte";
+
+  const dispatch = createEventDispatcher<{ search: SelectOption }>();
+
+  export let name: string;
+  export let id = `${name}-label`;
```

Comment:
(Nit) Pourquoi pas... ?

```suggestion
  export let id = name;
```